### ISSUE TITLE
Can't Find Dolphin Process With Linux Version

### ISSUE THREAD

**Issue by KingJigglypuff** [aldelaro5/Dolphin-memory-engine#64]
As the title describes, the Linux release is failing to find the Dolphin process. I haven't tried with mainline, but I'm trying to use it with the appimage provided by the [Project+ website](https://github.com/jlambert360/FPM-AppImage/releases/tag/v2.3.1)

I'm not sure if it'll help, but my Linux OS is Mint Cinnamon.

**Comment by aldelaro5** [comment]
So, the process detection goes by the executable's name which is normally "dolphin-emu". If it differs from that, it could cause the detection to fail.

I would make sure if you check your process's list that the name is indeed "dolphin-emu" and if not, you could rename the executable file so that it matches that.

**Comment by KingJigglypuff** [comment]
I renamed the appimage file to "dolphin-emu", and I got closer with that. DME now registers the Dolphin process, but it still can't hook, as it doesn't register that an emulation has started.

![image](https://user-images.githubusercontent.com/23140477/191414284-fc600e11-44f8-4d8d-a3c5-473be562e2cd.png)

**Comment by aldelaro5** [comment]
Have you tried the troublrshooting steps outlined at https://github.com/aldelaro5/Dolphin-memory-engine#troubleshouting ?

This tends to be common on linux.

**Comment by KingJigglypuff** [comment]
I tried that, and it didn't work for me to say the least.

However, what did work for me was adding my process' name to the LinuxDolphinProcess.cpp file (Source/DolphinProcess/Linux), and building from source. Since it turned out that even though I renamed the appimage to dolphin-emu, the process itself never changed, so the solution was to just add the process name.

If you want to update it on your end, just add a process reference for "AppRun". I'm sorry for the trouble, I should have honestly thought of this sooner and did a bit more research.
